### ISSUE TITLE
[Cop idea] DISPLAY endpoint above controller actions

### ISSUE THREAD

**Issue by anthony-robin** [rubocop-hq/rubocop#5560]
That could be a good idea to add a cop that would ensure there is 
a comment that display the endpoint corresponding to a controller
action like Rails append when scaffolding a resource.

Example:

```ruby
# good
class PostsController < ApplicationController
  # GET /posts
  def index
  end
end

# bad
class PostsController < ApplicationController
  def index
  end
end
```

What do you think about it ?

**Comment by mikegee** [comment]
I wouldn't want this cop enabled on my projects, but that doesn't mean it shouldn't exist. Maybe it could be configured to enforce the "bad" examples on my controllers?

This seems like a tricky cop to implement since Rubocop typically only has the current file context while scanning, but this cop would need to read from `config/routes.rb` too. (Maybe just make assumptions about the 7 typical REST endpoints instead?) Fun project if you wanna give it a shot!

**Comment by Drenmi** [comment]
Been thinking about this myself, but I ended up submitting a feature request with my IDE provider instead, since comments tend to never get updated with changes. 🙂 Not opposed to the cop though.

**Comment by bbatsov** [comment]
I'm closing this ticket due to no recent activity. Feel free to re-open it if you ever come back to it.
